apps/storybook/src/LineVis.stories.tsx

```diff
@@ -31,114 +29,17 @@ const errorsArray = ndarray(
   dataArray.shape
 );
 
-const Template: Story<LineVisProps> = (args) => {
-  const {
-    dataArray,
-    domain: storyDomain,
-    scaleType,
-    errorsArray,
-    auxiliaries,
-  } = args;
-
-  // If story doesn't provide `domain`, compute it automatically
-  const auxArrays = (auxiliaries || []).map(({ array }) => array);
-  const domain =
-    storyDomain ||
-    getCombinedDomain([
-      getDomain(dataArray, scaleType, errorsArray),
-      ...getDomains(auxArrays, scaleType),
-    ]);
-
-  return <LineVis {...args} domain={domain} />;
-};
-
-export const Default = Template.bind({});
-Default.args = {
-  dataArray,
-  domain,
-};
-
-export const Domain = Template.bind({});
-Domain.args = {
-  dataArray,
-  domain: [-100, 100],
-};
-
-export const Abscissas = Template.bind({});
-Abscissas.args = {
-  dataArray,
-  domain,
-  abscissaParams: {
-    value: abscissas,
-  },
-};
-
-export const DescendingAbscissas = Template.bind({});
-DescendingAbscissas.args = {
-  dataArray,
-  domain,
-  abscissaParams: {
-    value: [...abscissas].reverse(),
-  },
-};
-
-export const ErrorBars = Template.bind({});
-ErrorBars.args = {
-  dataArray,
-  errorsArray,
-  showErrors: true,
-};
-
-export const AuxiliaryArrays = Template.bind({});
-AuxiliaryArrays.args = {
-  dataArray: primaryArray,
-  auxiliaries: [{ array: secondaryArray }, { array: tertiaryArray }],
-};
-
-export const AuxiliaryLabels = Template.bind({});
-AuxiliaryLabels.args = {
```

Comment:
This felt redundant. I've moved the labels into the `AuxiliaryArrays` story. I guess we no longer demonstrate what the tooltip looks like when there are no labels, but I don't think that's very interesting.